docs/en/sticky_keys.md

```diff
@@ -40,7 +40,7 @@ KC.SK(
 ### `defer_release`
 
 If `False` (default): release sticky key after the first interrupting key
-releases.
+is released, or another is pressed. THis PRevents TYpos LIke THese.
```

Comment:
Are you certain about the `another is pressed` part? Neither code nor unit test suggest that.

Reply:
Yes. Will unit test for that then as well as for the issue in Zulip.